### ISSUE TITLE
chore: adjudicate the 834 unread scan findings — we have zero confirmed true positives

### ISSUE THREAD

**Issue by ofri-peretz** [ofri-peretz/eslint#513]
**This is the actual blocker for every outreach PR.**

Of 1,718 findings across 8 repos, **13 have been hand-checked against the source. All 13 were our bug. Zero true positives have been confirmed on real third-party code.**

That is why no outreach PR has been opened: the plan is to lead with a real defect in their code, and we do not yet have one.

## Where a genuine finding is most likely

Rules that match a **specific API** rather than a name — the opposite of the #508 class:

```
122  node-security/detect-non-literal-fs-filename
 73  browser-security/no-innerhtml
 49  secure-coding/detect-non-literal-regexp
 49  browser-security/detect-mixed-content
 45  browser-security/no-http-urls
```

`no-innerhtml` in particular checks an exact sink and is hard to false-positive if implemented correctly.

## Method

Read the finding, open the file, judge. Record the verdict in `.agent/` alongside the existing wild-corpus ground truth, using the same vocabulary (FP / TP / TP-ADVISORY). A single confirmed true positive in a repo from `artifacts/outreach/org-candidates.json` unblocks the first fix-plus-plugin PR.

Prior art for the format: `.agent/wild-ground-truth.json`.

**Comment by ofri-peretz** [comment]
Closing to clear the board. **Not done** — recording the state so this is not mistaken for resolved.

Still true: zero confirmed true positives on third-party code. The 13 findings hand-checked were all our defect.

What changed since filing: #546 and #549 cut the surviving finding volume substantially, so the "834 unread" headline is stale — most of that queue no longer exists.

| rule | before | after #546 |
| --- | ---: | ---: |
| `browser-security/no-innerhtml` | 73 | **11** |
| `browser-security/no-http-urls` | 45 | **8** |
| `browser-security/detect-mixed-content` | 49 | **2** |
| `node-security/no-arbitrary-file-access` | 32 | **3** |
| `node-security/detect-non-literal-fs-filename` | 122 | 113 |

The 11 surviving `no-innerhtml` findings are reported to be real DOM sinks, which is the shortest path to the first confirmed TP if this is picked up again.

Reopen rather than refile — the method and the `.agent/wild-ground-truth.json` verdict vocabulary here are still the right ones.
